config/routes.rb

```diff
@@ -3,6 +3,8 @@
 
 Rails.application.routes.draw do
   devise_for :'admin/users', ActiveAdmin::Devise.config
+  # get 'admin/acas_searches/*id', as: :admin_acas_search_id, to: 'admin/acas_searches#show'
+  # get 'admin/acas_searches', as: :admin_acas_search, to: 'admin/acas_searches#show'
```

Comment:
Should this line and the one above be commented out?

Reply:
Ill remove now - thanks